support/cas-server-support-pac4j-core-clients/src/test/java/org/apereo/cas/support/pac4j/authentication/DelegatedClientFactoryTests.java

```diff
@@ -79,13 +80,18 @@ public void verifyFactoryForSamlClients() throws Exception {
         saml.setServiceProviderMetadataPath(new File(FileUtils.getTempDirectoryPath(), "sp.xml").getCanonicalPath());
         saml.setServiceProviderEntityId("test-entityid");
         saml.setForceKeystoreGeneration(true);
+        saml.setMessageStoreFactory("org.pac4j.saml.store.HttpSessionStoreFactory");
```

Comment:
`HttpSessionStoreFactory.class.getName()`

Use the class itself to get its name, so if it moves or is deleted, we can catch the issue at compile-time.